tensorflow/python/ops/data_flow_ops.py

```diff
@@ -2099,6 +2100,10 @@ def __init__(self,
       file_pattern: File path to the dataset, possibly containing wildcards.
         All matching files will be iterated over each epoch.
       batch_size: How many records to return at a time.
+      minibatches: None by default, otherwise specifies
```

Comment:
I was debating between the two and just picked one, I was thinking you choose a batch size because that is what will fit on your machine(s), then subdivide it.